sharktank/sharktank/utils/export_artifacts.py

```diff
@@ -202,19 +202,29 @@ def compile_to_vmfb(
             f"{mlir_path}",
             f"--iree-hip-target={self.iree_hip_target}",
             f"--iree-hal-target-backends={self.iree_hal_target_backends}",
+            "--iree-dispatch-creation-enable-aggressive-fusion=true",
+            "--iree-global-opt-propagate-transposes=true",
+            "--iree-opt-aggressively-propagate-transposes=true",
+            "--iree-opt-data-tiling=false",
+            '--iree-preprocessing-pass-pipeline="builtin.module\\(util.func\\(iree-preprocessing-generalize-linalg-matmul-experimental\\)\\)"',
```

Comment:
This `export_artifacts` script looks generic. Don't add program-specific flags to a generic script, much less "experimental" flags. If an individual test wants to opt in to flags, add them to the test (e.g. by allowing extra args to be passed to this function).